packages/integration-sdk-runtime/src/execution/dependencyGraph.ts

```diff
@@ -75,6 +75,7 @@ export function buildStepDependencyGraph<
  * created more leaf nodes and executes them. This continues
  * until there are no more nodes to execute.
  */
+
```

Comment:
nitpick: shouldn't be a space since this doc string is related to below function?